source/api.rst

```diff
@@ -18,6 +18,8 @@ Tokens have to be generated from the **EGI DataHub** (Onezone) interface as
 documented in :ref:`auth-token-using-web-interface` or using a command line
 call as documented hereafter.
 
+Bear in mind that a unique API tokens can be used with both Onezone and Oneprovider.
```

Comment:
s/tokens/token/

Reply:
Also I would rather use "single" instead of "unique"

Reply:
Agreed.